packages/pools/src/router/route.ts

```diff
@@ -13,43 +13,76 @@ export interface Route {
   tokenInDenom: string;
 }
 
-/** Throws if route is invalid */
-export function validateRoute(route: Route) {
+/** Single path through pools, with the initial amount calculated. */
```

Comment:
Could you expand this comment a little bit more 🙏 , I'm confused as to whether this refers to a method or the interface